docs/guides/filters.rst

```diff
@@ -26,6 +27,26 @@ is passed as parameter ``filter_by`` in the subscription.
         print(f'Received a photo of type {kwargs.get("type")}')
 
 
+`filter_expression` parameter
```

Comment:
I wonder whether we should be calling this `filter_expression`. We are exploring asyncapi compatible naming for things like payload and attributes. This means staying away from gcloud specific naming. So instead we could name it `cloud_filter_by`

Reply:
I'd like something more like `backend_filter_by` as we can implement more backends in the future and it is a naming well established.

Reply:
mmm, my mind was thinking more in `local_filter` vs `cloud_filter`. However if we think about a redis or rabbitmq broker, "cloud" wouldn't make sense. What about `broker_filter_by`?